sdk/@launchdarkly/launchdarkly_flutter_observability/lib/src/plugin/observability_plugin.dart

```diff
@@ -66,10 +68,15 @@ final class _ObservabilityHook extends Hook {
 
 /// LaunchDarkly Observability plugin.
 final class ObservabilityPlugin extends Plugin {
+  final List<Instrumentation> _instrumentations = [];
   final PluginMetadata _metadata = const PluginMetadata(
     name: _launchDarklyObservabilityPluginName,
   );
 
+  ObservabilityPlugin() {
+    _instrumentations.add(LifecycleInstrumentation());
+  }
```

Comment:
At some point we will want a cleanup ability, but for now just make the instrumentations.